Geometry/VeryForwardGeometryBuilder/test/print_geometry_info_cfg.py

```diff
@@ -11,14 +11,15 @@
 )
 
 # geometry
-process.load("Geometry.VeryForwardGeometry.geometryRP_cfi")
+process.load("Geometry.VeryForwardGeometry.geometryRPFromDD_2017_cfi")
 
 # load alignment correction
-process.load("Geometry.VeryForwardGeometryBuilder.ctppsIncludeAlignments_cfi")
-process.ctppsIncludeAlignments.RealFiles = cms.vstring("Alignment/CTPPS/data/RPixGeometryCorrections.xml")
+process.load("Geometry.VeryForwardGeometryBuilder.ctppsIncludeAlignmentsFromXML_cfi")
+process.ctppsIncludeAlignmentsFromXML.RealFiles = cms.vstring("Geometry/VeryForwardGeometryBuilder/test/sample_alignment_corrections.xml")
```

Comment:
what are the plans to have this in DB?

Reply:
Actually we would have liked to have it already, but we had to postpone it due to other urgencies. We are now planning to have it for the next release, probbaly 10_2 if not before.